sessionctx/stmtctx/stmtctx.go

```diff
@@ -120,8 +120,7 @@ type StatementContext struct {
 	RuntimeStatsColl *execdetails.RuntimeStatsColl
 	TableIDs         []int64
 	IndexIDs         []int64
-	NowTs            time.Time
-	SysTs            time.Time
```

Comment:
It seems that SysTs are affected by the timestamp variable, and NowTs are not. I am not sure if I will remove it directly.

Reply:
SysTs is only used in function evalNowWithFsp(calculating "now/current_timestamp" function call),  nowTs is used more places(curDate, utc_timestamp). seems we need only one to store the statement "currenti time"